system-int-test/test/cmr/system_int_test/search/autocomplete_collection_facet_search_test.clj

```diff
@@ -27,64 +27,103 @@
                      [(ingest/reset-fixture)
                       autocomplete-fixture]))
 
-(deftest autocomplete-suggest-test
-  (testing "returned result form test"
-   (let [response (search/get-autocomplete-suggestions "foo")
-         body (:body response)
-         data (json/parse-string body true)]
-     (is (contains? data :query))
-     (is (contains? data :results))
-     (is (contains? (:results data) :items))))
+(deftest autocomplete-response-test
+  (testing "response form test"
+     (let [response (search/get-autocomplete-suggestions "q=foo")
+           body (:body response)
+           data (json/parse-string body true)]
+       (is (contains? data :feed))
+       (is (contains? (:feed data) :entry))))
 
+  (testing "returns CMR-Hits header"
+     (let [response (search/get-autocomplete-suggestions "q=foo")
+           headers (:headers response)]
+       (is (:CMR-Hits headers))
+       (is (= "1" (:CMR-Hits headers))))))
+
+(deftest autocomplete-functionality-test
   (testing "partial value match search"
-   (let [response (search/get-autocomplete-suggestions "f")
+   (let [response (search/get-autocomplete-suggestions "q=f")
```

Comment:
You are basically running the same test over and over with different input and expected responses. Take a look at the `are3` macro used in other tests for a way to do this more concisely.